ada_2_fhir-r4/zibs2020/ada_instance/nl-core-FamilySituation-01.xml

```diff
@@ -0,0 +1,48 @@
+<gezinssituatie_registratie rights="update delete" owner="jornduwel" app="zib2020-nl"
+    shortName="gezinssituatie_registratie"
+    formName="gezinssituatie"
+    transactionRef="2.16.840.1.113883.2.4.3.11.60.121.4.2.7.13"
+    transactionEffectiveDate="2020-09-01T00:00:00"
+    versionDate=""
+    prefix="zib2017-"
+    language="nl-NL"
+    title="Gezinssituatie-eoi"
+    desc=""
+    id="4da9c9c9-98a6-4852-a9c9-c998a658522e">
+    <gezinssituatie conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.7.13.1">
+        <onderwerp conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.1">
+            <patient-id value="999911120" root="2.16.840.1.113883.2.4.6.3"
+                conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.2"/>
+        </onderwerp>
+        <burgerlijke_staat conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.7.13.7">
+            <burgerlijke_staat_rc value="BurgerlijkeStaat-eoi" datatype="reference"
```

Comment:
I do not think this can be handled any better:
- The whole 'referenties' block is specific to this project - so we cannot use this in our code.
- As with all other zibs, we expect the coder of the use case to handle what to do with (the input of) each zib. When FamilySituation and thus MaritalStatus is relevant to the use case, it should be expected that the coder handles this in the payload by calling template nl-core-Patient with param $maritalStatus (yes, this exists) correctly filled. 
- For this ""use case"" we ""handle"" it by using nl-core-Patient-bundled in our payload, which always includes _a_ (although not specifically _this_) marital status.
- In other ada2fhir cases we also just do a 'for-each problem' or 'for-each procedure' to make a reference withou. For-each maritalStatus just wouldn't make sense, and we always apply a subject anyway

Question is if this situation can be made any more clear. Outputting a warning perhaps?

Reply:
I didn't know the 'referenties' block is specific to this project!

You already include 'Reference to burgerlijke_staat is handled by including it in the Patient' in the nl-core-FamilySituation template, so that might be clear enough.